src/purview.ts

```diff
@@ -446,7 +280,7 @@ async function handleMessage(
 
       const cssStateID = message.cssStateID
       if (cssStateID) {
-        const cssState = await reloadOptions.getCSSState(cssStateID)
+        const cssState = await Purview.reloadOptions.getCSSState(cssStateID)
```

Comment:
Because `reloadOptions` seems to need to live in the Purview namespace, this seemed like a sensible way to refactor these codes.